app/controllers/users_controller.rb

```diff
@@ -13,6 +13,36 @@ def index
 
   def show
     @user = User.find(params[:id])
+    @ratings = Rating.where(ratee_id: params[:id])
+    @rates = Rating.where(rater_id: params[:id])
+
+    @average_score = average_score(@user)
+    @average_rate = average_rate(@user)
+  end
+
+
+  def average_score(user)
+    if !@ratings.empty?
+      sum = 0
+      @ratings.each do |rating|
+        sum += rating.score
+      end
+      average = (sum/@ratings.count.to_f).round(2)
+    else
+      "N/A"
+    end
+  end
+
+  def average_rate(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `average_rate(*)` if you want the method to accept any arguments but don't care about them.